guides/TERMINOLOGY.md

```diff
@@ -0,0 +1,7 @@
+#Terminology
+- Product - A piece of software controlled by an operator (e.g. RHSSO, 3scale, etc)
+- Resource - Any CRD that is managed by the product's operator. 
```

Comment:
I don't think this should be here in my opinion, Resource is a common k8s term.

Reply:
cool, removed